Controller/Adminhtml/Index/Disconnect.php

```diff
@@ -75,7 +75,7 @@ public function validateForCsrf(RequestInterface $request): ?bool
      */
     public function execute()
     {
-        $this->publisher->publish('storekeeper.disconnect.events', 'sk_disconnect');
+        $this->publisher->publish('storekeeper.queue.events', '{"type":"disconnect"}');
```

Comment:
```suggestion
        $this->publisher->publish('storekeeper.queue.events', json_encode(["type" =>"disconnect"]));
```
I'm willing to sacrifice some mili seconds for for having an any parse errors.